padertorch/train/hooks.py

```diff
@@ -168,6 +168,8 @@ def empty_summary_dict():
             texts=dict(),
             figures=dict(),
             timings=dict(),
+            buffers=dict(),
+            snapshots=dict()
```

Comment:
Can you add a few comments, what buffers and snapshots are?

Reply:
I wrote a line on the two keys in the SummaryHook description. I think further documentary of those two keys should be placed inside the ModifySummary function.

Reply:
Yes that would be a good place. Can you add there some sentence to describe these keys?
An assert in that function would also be nice, because then the user knows where to change it.

Is it possible to write a small test for buffers and snapshots? A "burn" test would be enough.

Reply:
I will commit those in an additional issue, as the updates for hooks where not meant to be in this pull request, anyway.